source/usage.rst

```diff
@@ -0,0 +1,47 @@
+Usage
+===========
+
+The Compiler has some command line options:
+
+.. code:: bash
+
+    Usage: manimdsl [-hpV] [-o=<output>] [-q=<quality>] <file>
+    ManimDSL compiler to produce manim animations
+          <file>                The manimdsl file to compile and animate
+      -h, --help                Show this help message and exit.
+      -o, --output=<output>     The animated mp4 file location (default: out.mp4)
+      -p, --python              Output generated python & manim code (optional)
+      -q, --quality=<quality>   Quality of animation. [low, high] (default: low)
+      -V, --version             Print version information and exit.
+
+
+Command Line Arguments
+----------------------
+
+<file>
+~~~~~~~~~~~~~~~~~~~~~
+
+ - The ``.manimdsl`` file to compile
+
+-o=<output>, --output=<output>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - The path to where the MP4 file should be saved (defaults to ``out.mp4``)
+ - If the path does not exist, all the subdirectories will be created
```

Comment:
'All the subdirectories specified in the path will be created'? Just for clarity